src/Command/CleanIndexCommand.php

```diff
@@ -59,7 +59,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return 0;
         }
 
-        $lockAcquired = $this->locker->lockCommand($this->getName());
+        $lockAcquired = $this->locker->lockCommand(__CLASS__);
```

Comment:
The lock name should be <=64 chars, but I guess this works with all the current class names.. so probably fine. `(string)$this->getName()` would work too though.